test/VersionInfo/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Taef" version="10.75.221207001" targetFramework="native" />
-  <package id="Microsoft.Windows.CppWinRT" version="2.0.221121.5" targetFramework="native" />
+  <package id="Microsoft.Taef" version="10.82.230714001" targetFramework="native" />
```

Comment:
Is there a "repo wide packages config" that we should be using here? Like a top-level "directory.props" or something that can be included?

Reply:
Unfortunately no. VS supports that for *proj files (and thus C# since they only reference packages in *.csproj), but for C++ VS requires Nuget's packages.config and that has no macro or #include mechanism. That's why WinAppSDK built our own.

Dependencies MUST be listed in [eng\Version.Details.xml](https://github.com/microsoft/WindowsAppSDK/blob/main/eng/Version.Details.xml) if Maestro should automagically update when newer versions are available or [eng\Version.Dependencies.xml](https://github.com/microsoft/WindowsAppSDK/blob/main/eng/Version.Dependencies.xml) for manual updates only.

After editing those, run `DevCheck.cmd -CheckDependencies` to verify everything in the repo uses only sanctioned dependencies and only the sanctioned versions, or `DevCheck.cmd -SyncDependencies` to update everything in the repo as necessary.

The build pipelines include a `-CheckDependencies` step so you can't commit anything not matching eng\Version.*.xml.

This -Check/SyncDependencies was introduced late last year after repeated